README.rst

```diff
@@ -52,15 +52,17 @@ for us, which you can activate via::
 
 See the `pipenv documentation <https://docs.pipenv.org/>`_ for more detail.
 
+Alternatively, you can used a standard python virtualenv if you prefer.
```

Comment:
used -> use?

Reply:
Geez, I read this like 3 times before submitting it and STILL had a typo!  Thanks, I will fix it.